core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -324,8 +328,23 @@ protected void doStop() throws Exception {
     StatisticsHandler statsHandler = new StatisticsHandler();
     statsHandler.setHandler(handlers);
 
-    server.setHandler(statsHandler);
-
+    final ServletContextHandler webSocketServletContext =
+        new ServletContextHandler(ServletContextHandler.SESSIONS);
+    webSocketServletContext.setContextPath("/ws");
+    final ContextHandlerCollection contexts = new ContextHandlerCollection();
+    contexts.setHandlers(new Handler[] {
+        webSocketServletContext,
+        statsHandler
+    });
+
+    server.setHandler(contexts);
+    try {
+      ServerContainer container =
+          WebSocketServerContainerInitializer.configureContext(webSocketServletContext);
+      registerWebSocketEndpoints(container);
+    } catch (ServletException e) {
+      log.error("unable to configure sebsockets", e);
```

Comment:
sebsockets

Reply:
also, if this happens is logging and swallowing the right thing to do or just pass the exception up? letting it propagate seems like it is better unless there are expected cases where we'd try to set this up and but it would be expected to fail.